BrightCoders-Proyectos/tribo-mobile
# Feature/15 onboarding screen UI

# Description
What did you implemented/Why did you implemented this:
 - I created three screens files that for now will show how the On Boarding screens should look like based on issue #15 design.
 - It was designed and coded in order to start looking for components like the screen indicator in OnBoardScreenConect and OnBoardScreenCall (Will be implemented soon in sprint 2 after adding the react-navigation)
 - I also fixed some small details in the color file and implemented in the readme screens.
 
 # Testing
1. Download my branch (feature/15-Onboarding-Screen-UI).
2. Go to "tribomobile" folder.
3. If you do not see the modules : npm install.
3.  npx react-native start.
4. npx react-native run-android
5. You will se as default the OnBoardScreenConect in the app, if you want to see the others just change their import name like "OnBoardScreenCall" or "OnBoardScreenReceive".

##Screenshots:
-------------------------------------------------- Resulted Design -------------------------------------------------- 
![image](https://user-images.githubusercontent.com/69820530/102556053-9e940800-408d-11eb-95b9-bd0cc084add1.png)
![image](https://user-images.githubusercontent.com/69820530/102555769-0e55c300-408d-11eb-9772-04f6f81d3096.png)
![image](https://user-images.githubusercontent.com/69820530/102555800-1f9ecf80-408d-11eb-8a82-3ad315906525.png)
